docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+services:
+  backend:
+    build: 
+      context: ./backend
+      dockerfile: Dockerfile.dev
+    container_name: backend
+    volumes:
+      - ./backend/app:/districtr-backend/app   # Adjust volumes to point to the backend folder
+      - ./backend/scripts:/districtr-backend/scripts  # Ensure scripts are mounted from backend directory
+      - ./sample_data:/districtr-backend/sample_data
+    env_file:
+      - ./backend/.env  # Ensure the env file is also from the backend folder
```

Comment:
I: Rather than using `./backend/.env` what if we used `environment:` or created a `.env.docker` or something?

Reply:
Done!